python/lsst/pex/exceptions/wrappers.py

```diff
@@ -5,6 +5,11 @@
 from future.utils import with_metaclass
 from . import exceptions
 
+__all__ = ["register", "ExceptionMeta", "Exception", "LogicError",
```

Comment:
I think `__all__` goes above the imports https://developer.lsst.io/python/style.html#standard-code-order-should-be-followed